signer/src/transaction_coordinator.rs

```diff
@@ -418,14 +419,29 @@ where
             tracing::debug!("no requests to handle, exiting");
             return Ok(());
         };
-
         tracing::debug!(
             num_deposits = %pending_requests.deposits.len(),
             num_withdrawals = pending_requests.withdrawals.len(),
             "fetched requests"
         );
         // Construct the transaction package and store it in the database.
         let transaction_package = pending_requests.construct_transactions()?;
+        // Get the requests from the transaction package because they have been split into
+        // multiple transactions.
+        let sbtc_requests = BitcoinPreSignRequest {
+            requests: transaction_package
+                .iter()
+                .map(|tx| (&tx.requests).into())
+                .collect(),
+            fee_rate: pending_requests.signer_state.fee_rate,
+            last_fees: pending_requests.signer_state.last_fees.map(Into::into),
+        };
+
+        // Share the list of requests with the signers.
+        self.send_message(sbtc_requests, bitcoin_chain_tip).await?;
+        // Wait to reduce chance that the other signers will receive the subsequent
+        // messages before the BitcoinPreSignRequest one.
+        tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
```

Comment:
Given the variability of our network, we're probably going to want to make this a configurable parameter with a default on testnet/mainnet that is ~5-10 seconds or so. For devnet and local testing we can keep this at 1 second.

Reply:
Well, I'm not sure if we want to do the intermediate work and make this configurable or go straight for having the coordinator wait for `Ack`s. Your call.

Reply:
done in https://github.com/stacks-network/sbtc/pull/905/commits/468153f8dcb67255e836ccdfe65d4e449fd63993 . 
I should have done it in a separate branch, sorry, my mistake!